apps/voting/app/src/components/DisputableStatusLabel.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  GU,
+  IconAttention,
+  IconClock,
+  IconClosed,
+  IconInfo,
+  Tag,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { DISPUTABLE_VOTE_STATUSES } from '../disputable-vote-statuses'
+
+function DisputableStatusLabel({ status }) {
+  const theme = useTheme()
+  const statusAttributes = new Map([
+    [
+      DISPUTABLE_VOTE_STATUSES.get('Active'),
+      {
+        background: null,
+        label: 'Scheduled',
+        Icon: IconClock,
+        color: null,
+      },
+    ],
+    [
+      DISPUTABLE_VOTE_STATUSES.get('Cancelled'),
+      {
+        background: String(theme.surfaceUnder),
+        label: 'Cancelled',
+        Icon: IconClosed,
+        color: String(theme.disabledContent),
+      },
+    ],
```

Comment:
We're on the right track! I think we could simplify this even more though! How about we provide a function outside of the component without the string conversions? because we've already filtered the `Symbol`s we can use them as keys directly.

something like:

```
function getAttributes(status, theme) {
  const attributes = {
    [VOTE_STATUS_ACTIVE]: {
      label: 'Scheduled',
      Icon: IconClock,
    },
    [VOTE_STATUS_PAUSED]: {
      background: theme.warningSurface,
      color: theme.warningSurfaceContent,
      label: 'Challenged',
      Icon: IconAttention,
    },
    ....
  }

  return attributes[status]
}

```